src/pages/library/DataBrowser.js

```diff
@@ -47,21 +47,21 @@ const extractCatalogFilters = dataCatalog => {
     labels: _.values(datasetAccessTypes),
     labelRenderer: accessValue => {
       const lowerKey = _.toLower(accessValue)
-      const iconKey = accessValue === datasetAccessTypes.GRANTED ? 'unlock' : 'lock'
+      const iconKey = accessValue === datasetAccessTypes.Granted ? 'unlock' : 'lock'
       return [div({ key: `access-filter-${lowerKey}`, style: { display: 'flex' } }, [
         icon(iconKey, { style: { color: styles.access[lowerKey], marginRight: 5 } }),
         div([accessValue])
       ])]
     }
   }, {
     name: 'Consortium',
-    labels: getUnique(dataset => getConsortiumsFromDataset(dataset), dataCatalog)
+    labels: getUnique(dataset => getConsortiumTitlesFromDataset(dataset), dataCatalog)
   }, {
     name: 'Data use policy',
     labels: getUnique(dataset => dataset['TerraDCAT_ap:hasDataUsePermission'], dataCatalog),
     labelRenderer: rawPolicy => {
       return [div({ key: rawPolicy, style: { display: 'flex', flexDirection: 'column' } }, [
-        h(DatasetReleasePolicyDisplayInformation, { dataUsePermission: rawPolicy })
+        h(DatasetReleasePolicyDisplayInformation, { 'TerraDCAT_ap:hasDataUsePermission': rawPolicy })
```

Comment:
When this uses `TerraDCAT_ap:hasDataUsePermission`, is it different than `DatasetResponse.'TerraDCAT_ap:hasDataUsePermission '`? If it's the same thing, I wonder if there's a way to link the two together. If it's a different thing, then maybe it'd be better to use a different name to avoid confusion.

Reply:
We want it to be the same thing, because `DatasetReleasePolicyDisplayInformation` is designed to take in a dataset, however because of our filter implementation, we only get the raw object to filter by, so we are limited with what is in scope here.

There's probably some refactors of the filters we want to do, but I think that is out of scope of this PR.